---
title: 'Deconer: An Evaluation Toolkit for Reference-based Deconvolution Methods Using
  Gene Expression Data'

# Authors
# A YAML list of author names
# If you created a profile for a user (e.g. the default `admin` user at `content/authors/admin/`), 
# write the username (folder name) here, and it will be replaced with their full name and linked to their profile.
authors:
- Wei Zhang
- Xianglin Zhang
- Qiao Liu
- Lei Wei
- Xu Qiao
- Rui Gao
- Zhiping Liu
- Xiaowo Wang

# Author notes (such as 'Equal Contribution')
# A YAML list of notes for each author in the above `authors` list
author_notes: []

date: '2025-02-01'

# Date to publish webpage (NOT necessarily Bibtex publication's date).
publishDate: '2025-08-14T16:46:40.369927Z'

# Publication type.
# A single CSL publication type but formatted as a YAML list (for Hugo requirements).
publication_types:
- article-journal

# Publication name and optional abbreviated publication name.
publication: '*Genomics, Proteomics & Bioinformatics*'
publication_short: ''

doi: 10.1093/gpbjnl/qzaf009

abstract: In recent years, computational methods for quantifying cell-type proportions
  from transcription data have gained significant attention, particularly those reference-based
  methods which have demonstrated high accuracy. However, there is currently a lack
  of comprehensive evaluation and guidance for available reference-based deconvolution
  methods in cell-type deconvolution analysis. In this study, we introduce Deconvolution
  Evaluator (Deconer), a comprehensive toolkit for the evaluation of reference-based
  deconvolution methods. Deconer provides various simulated and real gene expression
  datasets, including both bulk and single-cell sequencing data, and offers multiple
  visualization interfaces. By utilizing Deconer, we conducted systematic comparisons
  of 16 reference-based deconvolution methods from different perspectives, including
  method robustness, accuracy in deconvolving rare components, signature gene selection
  performance, and external reference construction capability. We also performed an
  in-depth analysis of the application scenarios and challenges in cell-type deconvolution
  methods. Finally, we provided constructive suggestions for users to select and develop
  cell-type deconvolution algorithms. This study provides novel insights for researchers,
  assisting them in choosing appropriate toolkits, applying solutions in clinical
  contexts, and advancing the development of deconvolution tools tailored to gene
  expression data. The tutorials, manual, source code, and demo data of Deconer are
  publicly available at https://honchkrow.github.io/Deconer/ and https://ngdc.cncb.ac.cn/biocode/tool/7577.

# Summary. An optional shortened abstract.
summary: ''

tags: []

# Display this page in a list of Featured pages?
featured: false

# Links
url_pdf: ''
url_code: ''
url_dataset: ''
url_poster: ''
url_project: ''
url_slides: ''
url_source: ''
url_video: ''

# Custom links (uncomment lines below)
# links:
# - name: Custom Link
#   url: http://example.org

# Publication image
# Add an image named `featured.jpg/png` to your page's folder then add a caption below.
image:
  caption: ''
  focal_point: ''
  preview_only: false

# Associated Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects: ['internal-project']` links to `content/project/internal-project/index.md`.
#   Otherwise, set `projects: []`.
projects: []
links:
- name: URL
  url: https://doi.org/10.1093/gpbjnl/qzaf009
---

Add the **full text** or **supplementary notes** for the publication here using Markdown formatting.
